airbyte-integrations/connectors/source-file-secure/metadata.yaml

```diff
@@ -2,10 +2,12 @@ data:
   allowedHosts:
     hosts:
       - "*"
+  connectorBuildOptions:
+    baseImage: airbyte/python-connector-base:1.0.0
   connectorSubtype: file
   connectorType: source
   definitionId: 778daa7c-feaf-4db6-96f3-70fd645acc77
-  dockerImageTag: 0.3.12
```

Comment:
nit: why a minor bump and not a patch?

Reply:
I consider its a minor because we change something quite fundamental: the connector base image.
But from a user perspective it's not providing a new feature.
What's your opinion?

Reply:
Generally I think specially as we make versions more user-facing, the versioning should be expressed as such. I think this is what's laid out in the guidelines here https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook/#semantic-versioning-for-connectors

However, I do agree changing the base image might be a fundamental change, so... 🤷 Up to you!